core/embed/trezorhal/stm32f4/touch/ft6x36.c

```diff
@@ -262,7 +262,9 @@ uint32_t touch_read(void) {
   if (touch_data[1] == GESTURE_NO_GESTURE) {
     xy = touch_pack_xy((X_POS_MSB << 8) | X_POS_LSB,
                        (Y_POS_MSB << 8) | Y_POS_LSB);
-    if ((number_of_touch_points == 1) && (event_flag == EVENT_PRESS_DOWN)) {
```

Comment:
I actually cannot see a functional difference between the two variants, seems to me that both are doing the same change - adding a start event in case `EVENT_CONTACT` and not previously `touching`, or am i missing something?

Reply:
using your version anyway as it seems more readable
https://github.com/trezor/trezor-firmware/commit/ddaf625fb40c21d0e5614b3c8b47c2d536f2b467

Reply:
I mean the previous version. Lets keep the latest one for more thorough evaluation then.